pubs/content.py

```diff
@@ -52,16 +52,17 @@ def _open(path, mode):
     else:
```

Comment:
I should have called it PEP8 ;)

Is this comment related in any way to the line it's attached to?